### PR TITLE
[doc]fix version 2.1.0 has invalid internal hyperlinks on the CREATE-TABLE page

### LINKED ISSUE #832: Version 2.1.0 has invalid internal hyperlinks on the CREATE-TABLE page
Version 2.1.0 has invalid internal hyperlinks on the CREATE-TABLE page
图片

### PR DESCRIPTION
**Pull request by cjj2010**
close #832

### CODE REVIEW

**Review by shasevendow — APPROVED** [review]
LGTM

**Comment by shasevendow** [comment]
Please update 2.1 version documents as well.

**Comment by cjj2010** [comment]
> Please update 2.1 version documents as well.

It's necessary, it's already taken care of now

**Review by KassieZ — APPROVED** [review]

File: docs/sql-manual/sql-statements/Data-Definition-Statements/Create/CREATE-TABLE.md
Comment (KassieZ) [inline docs/sql-manual/sql-statements/Data-Definition-Statements/Create/CREATE-TABLE.md:1]: please alos fix version 3.0 and dev

**Pull request merged**